packages/shared/providers.yaml

```diff
@@ -5339,3 +5339,27 @@ falai:
             type: string
             title: API Key
             description: The API key for your fal.ai account
+
+acuity-scheduling:
+    display_name: Acuity Scheduling
+    categories:
+        - scheduling
+        - appointments
+    auth_mode: OAUTH2
+    authorization_url: https://acuityscheduling.com/oauth2/authorize
+    token_url: https://acuityscheduling.com/oauth2/token
+    connection_configuration:
+        - userId
+        - apiKey
+    post_connection_script: acuityPostConnection
+    webhook_routing_script: acuityWebhookRouting
+    proxy:
+        base_url: https://acuityscheduling.com/api/v1
+        decompress: true
+        paginate:
+            type: cursor
+            cursor_path_in_response: pagination.nextCursor
+            limit_name_in_request: limit
+            cursor_name_in_request: cursor
+            response_path: data
+    docs: https://developers.acuityscheduling.com/docs
```

Comment:
- Please use Nango's doc reference; `https://docs.nango.dev/integrations/all/acuity-scheduling`
- I have prepared a [logo](https://codefile.io/f/cQJ8xFe9gV) you can update with

Reply:
Ooo thanks, I was lowkey trying to figure out how I'll do logo